docs/guides/syntax.md

```diff
@@ -198,13 +198,21 @@ A suffix mapped to a [pseudo-element](https://developer.mozilla.org/en-US/docs/W
 -   `ph` for `::placeholder`
 -   `s` for `::selection`
 
+This example uses the `content` property to add open and closed quotes with the `::b` and `::a` pseudo-elements.
+
+```html
+<q class="Cnt(oq)::b Cnt(cq)::a">Hello world!</q>
```

Comment:
maybe do an example without `<q>` since that adds quotes :) use  `<blockquote>` perhaps

Reply:
Ya blockquote has default styles in our stylesheet, I can just use simple `<p>` tag